index.html

```diff
@@ -295,6 +295,7 @@ <h1 class="form-text">I'm always interested in hearing about new projects, so if
                 nestor.oro.alvarez@gmail.com
             </p>
         </footer>
+        <div id="overlay"></div>
```

Comment:
- [ ] Goog job adding the overlay as suggested by the previous reviewer, but kindly ensure you add the appropriate padding around the overlay to match the figma design 👍🏽 

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/216166069-4f6f5e9b-637c-4820-9eaa-dcd860bcc447.png)

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/216165959-9c038e19-22f0-4d62-97eb-a96ca46b05a6.png)